libbpfgo.go

```diff
@@ -1201,6 +1203,41 @@ func (b BPFProgType) String() (str string) {
 	return str
 }
 
+var BpfProgTypeMap = map[uint64]BPFProgType{
```

Comment:
I think this map should live in `helpers` next to the ParseBPFProgType() function, this way this map doesn't have to be exported.